src/app/profiles/store/profile.reducer.ts

```diff
@@ -0,0 +1,35 @@
+
+export interface ProfileFormState {
+  tv: string;
```

Comment:
Just naming, but what does the key `tv` actually represent? Consider adding a suffix or prefix to indicate the context of `tv`. For instance `tvProviders`, etc. Same goes with `streaming` and potentially `antenna`